viz.yaml

```diff
@@ -115,27 +118,113 @@ fetch:
     scripts: [scripts/fetch/map_data.R , scripts/fetch/utils.R]
     depends: viewbox_limits
 process:
+  -
+    id: precip_cells
+    location: cache/precip_cells.rds
+    reader: rds
+    cell_size: 80000
+    processor: cells_from_mask
+    scripts: [scripts/process/cells_from_mask.R]
+    depends: 
+      mask_poly: "state_map_data"
   - 
-    id: state_geoms_dataframe
-    location: cache/state_geoms_dataframe.rds
+    id: precip_polygons_svg
```

Comment:
Why did precip_cells become precip_polygons?

Reply:
I was trying to stay consistent w/ naming of the actual `sf` contents. States are polygons (and named that way) so I thought this made sense too.

Reply:
ahhh. makes sense.